code/modules/research/designs/misc.dm

```diff
@@ -42,7 +42,7 @@
 	name = "Mobile Operating Table"
 	desc = "A collapsed mobile operating table that can be carried around."
 	id = "surgery_rollerbed"
-	build_type = PROTOLATHE | MECHFAB
```

Comment:
sneaky

Reply:
oh wait, not just sneaky, you have a duplicate definition in designs/surgery.dm ???